### ISSUE TITLE
IE11 is not detected properly.

### ISSUE THREAD

**Issue by bwilson** [henadzit/grails-browser-detection#8]
userAgentIdentService.getBrowserVersion() does not work for IE11, and userAgentIdentService.getBrowser() returns "Mozilla". This is Microsoft's way of making IE pretend to be a browser that doesn't suck, but it would be great if the plugin could be made to detect IE11 properly.

**Comment by shri3k** [comment]
I agree. IE11 uses Trident/ as an identifier now rather than MSIE which it used in the past.
